deployments/examples/ocis_individual_services/.env

```diff
@@ -31,9 +31,19 @@ OCIS_JWT_SECRET=
 STORAGE_TRANSFER_SECRET=
 # Machine auth api key secret. Must be changed in order to have a secure oCIS. Defaults to "change-me-please"
 OCIS_MACHINE_AUTH_API_KEY=
+# System user machine auth api key. Must be changed in order to have a secure oCIS. Defaults to "change-me-please"
+SYSTEM_USER_API_KEY=
 # Number of services to run for extensions, that currently can be easily scaled. Defaults to 1.
 OCIS_SCALE=
 
+# Service user ids and a passwords, set to random strings, defaults to "changeme"
```

Comment:
How about going a bolt step and removing the "default secrets" totally? Then we also need to adapt the continuous deployment config

Reply:
You would need to provide an `.env` for each one then. I suggest we add them first.

Reply:
we could add the config options here, they will end up in the .env file: https://github.com/owncloud/ocis/blob/64df36066c2cc5b1ad38411fdb2b4f5c81b6ae75/deployments/continuous-deployment-config/ocis_individual_services/latest.yml#L36-L39